worlds/sc2/locations.py

```diff
@@ -789,6 +797,8 @@ def get_locations(world: Optional['SC2World']) -> Tuple[LocationData, ...]:
                      lambda state: logic.zerg_common_unit(state)),
         make_location_data(SC2Mission.DOMINATION.mission_name, "Northeast Baneling Nest", SC2HOTS_LOC_ID_OFFSET + 709, LocationType.EXTRA,
                      lambda state: logic.zerg_common_unit(state)),
+        make_location_data(SC2Mission.DOMINATION.mission_name, "Win Without 100 Eggs", SC2HOTS_LOC_ID_OFFSET + 710, LocationType.CHALLENGE,
+                     lambda state: logic.zerg_common_unit(state)),

```

Comment:
Winning without AA to deal with Mutas? I think it should be competent comp and competent AA (you need something more than just plain Hydras)

Reply:
Ah, I've done this exclusively with zerglings and banelings as you _can_ just ignore the mutas. But that's fair that this is not typical. Updating to `zerg_competent_comp` and `zerg_basic_anti_air`.